### PR TITLE
Core alert fields aliases

### PR DESCRIPTION
**Pull request by evisochek**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [x] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Initiator_PID.json [hunk 1]
```diff
@@ -28,5 +28,19 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
+<<<<<<< HEAD
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Initiator_PID.json:31]: Fix leftovers

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Initiator_PID.json [hunk 2]
```diff
@@ -28,5 +28,19 @@
...
+          "cliName": "carbonblackesprocessid",
+          "type": "shortText"
+        },
+        {
+            "cliName": "pid",
+            "type": "multiSelect"
+          }
+    ]
+=======
     "x2_fields": "actor_process_os_pid"
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Initiator_PID.json:44]: double

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Module.json [hunk 3]
```diff
@@ -28,5 +28,16 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
+<<<<<<< HEAD
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Module.json:31]: Fix leftovers

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Process_Execution_Signature.json [hunk 4]
```diff
@@ -28,5 +28,24 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
+<<<<<<< HEAD
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Process_Execution_Signature.json:31]: Fix leftovers

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Process_Execution_Signature.json [hunk 5]
```diff
@@ -28,5 +28,24 @@
...
+    "x2_fields": "action_process_signature_status",
+    "Aliases": [
+        {
+          "cliName": "f5silverlineattacksignaturename",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Process_Execution_Signature.json:35]: This field is for value such as ['Signed'] Vs not signed
Pretty sure these aliases are not for it

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Remote_IP.json [hunk 6]
```diff
@@ -53,6 +53,26 @@
...
+        {
+          "cliName": "deviceexternalip",
+          "type": "shortText"
+        },
+        {
+          "cliName": "externaladdresses",
+          "type": "shortText"
+        },
+        {
+          "cliName": "irondefensesrcip",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Remote_IP.json:74]: This is src

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Source_Zone_Name.json [hunk 7]
```diff
@@ -28,5 +28,24 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
+<<<<<<< HEAD
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Source_Zone_Name.json:31]: Clean up

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Target_Process_Name.json [hunk 8]
```diff
@@ -28,5 +28,28 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
+<<<<<<< HEAD
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Target_Process_Name.json:31]: Clean up

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Target_Process_Name.json [hunk 9]
```diff
@@ -28,5 +28,28 @@
...
+        {
+          "cliName": "childprocess",
+          "type": "multiSelect"
+        },
+        {
+            "cliName": "digitalguardianprocessname",
+            "type": "shortText"
+        },
+        {
+            "cliName": "processname",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Target_Process_Name.json:43]: Process name should be actor_process_image_name (Initiated By) not action_process_image_name (Target process name)

File: Packs/CoreAlertFields/IncidentFields/incidentfield-osparentcmd.json [hunk 10]
```diff
@@ -28,5 +28,16 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
+<<<<<<< HEAD
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-osparentcmd.json:31]: Clean up

File: Packs/CoreAlertFields/IncidentFields/incidentfield-osparentcmd.json [hunk 11]
```diff
@@ -28,5 +28,16 @@
...
+	"x2_fields": "os_actor_process_command_line",
+    "Aliases": [
+        {
+          "cliName": "parentcmdline",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-osparentcmd.json:35]: Parent cmd line should go to CGO CMD (causality_actor_process_command_line)

File: Packs/CoreAlertFields/IncidentFields/incidentfield-osparentname.json [hunk 12]
```diff
@@ -27,5 +27,16 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
+<<<<<<< HEAD
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-osparentname.json:30]: Clean up

File: Packs/CoreAlertFields/IncidentFields/incidentfield-osparentname.json [hunk 13]
```diff
@@ -27,5 +27,16 @@
...
+	"x2_fields": "os_actor_process_image_name",
+    "Aliases": [
+        {
+          "cliName": "parentprocess",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-osparentname.json:34]: Should go to CGO name (causality_actor_process_image_name)

File: Packs/CoreAlertFields/IncidentFields/incidentfield-ruleid.json [hunk 14]
```diff
@@ -28,5 +28,17 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
+<<<<<<< HEAD
+	"x2_fields": "matching_service_rule_id",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-ruleid.json:32]: Clean up

File: Packs/CoreAlertFields/IncidentFields/incidentfield-useragent.json [hunk 15]
```diff
@@ -27,5 +27,20 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
+<<<<<<< HEAD
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-useragent.json:30]: Clean up

**Review by David-BMS — CHANGES_REQUESTED** [review]

File: Packs/CoreAlertFields/IncidentFields/incidentfield-osparentname.json (hunk 13 above)
Comment (evisochek) [reply Packs/CoreAlertFields/IncidentFields/incidentfield-osparentname.json:33]: fixed

File: Packs/CoreAlertFields/IncidentFields/incidentfield-osparentcmd.json (hunk 11 above)
Comment (evisochek) [reply Packs/CoreAlertFields/IncidentFields/incidentfield-osparentcmd.json]: fixed

File: Packs/AzureActiveDirectory/ReleaseNotes/1_2_2.md [hunk 16]
```diff
@@ -0,0 +1,3 @@
+
+#### Incident Fields
+- **Azure Active Directory Identity and Access Detected Date Time**
```
Comment (David-BMS) [inline Packs/AzureActiveDirectory/ReleaseNotes/1_2_2.md:3]: @yaakovi @reutshal  This is the correct format? not line by line anymore?

File: Packs/CommonTypes/ReleaseNotes/3_2_9.md [hunk 17]
```diff
@@ -0,0 +1,42 @@
...
+- **OS**
+- **Dest OS**
+- **Detected Users**
+- **Command Line**
+- **Srcs**
+- **CMD line**
+- **Device Local IP**
+
+#### Indicator Fields
+- **MAC Address**
```
Comment (David-BMS) [inline Packs/CommonTypes/ReleaseNotes/3_2_9.md:42]: @evisochek Why was this changed?

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Process_Execution_Signature.json [hunk 18]
```diff
@@ -28,5 +28,17 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
-    "x2_fields": "action_process_signature_status"
+    "x2_fields": "action_process_signature_status",
+    "Aliases": [
+        
+        {
+            "cliName": "fireeyesignature",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Process_Execution_Signature.json:35]: This field indicators if a files is signed or not
Values are ['Signed'] Not Signed etc.

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Target_Process_Name.json [hunk 19]
```diff
@@ -28,5 +28,22 @@
...
     "fromVersion": "6.5.0",
-    "x2_fields": "action_process_image_name"
+    "x2_fields": "action_process_image_name",
+    "Aliases": [
+        {
+          "cliName": "childprocess",
+          "type": "multiSelect"
+        },
+        {
+            "cliName": "digitalguardianprocessname",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Target_Process_Name.json:38]: I think this refers to process and not child process 
So should be mapped to Initiated By(actor_process_image_name)

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Process_Execution_Signature.json (hunk 18 above)
Comment (evisochek) [reply Packs/CoreAlertFields/IncidentFields/incidentfield-Process_Execution_Signature.json:35]: fixed

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Initiated_By.json [hunk 20]
```diff
@@ -33,6 +33,21 @@
...
           "type": "shortText"
+        },
+        
+        {
+          "cliName": "carbonblackesprocessname",
+          "type": "shortText"
+        },
+        
+        {
+            "cliName": "digitalguardianparentprocessname",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Initiated_By.json:44]: This is CGO name (causality_actor_process_image_name)

File: Packs/CoreAlertFields/IncidentFields/incidentfield-Target_Process_Name.json [hunk 21]
```diff
@@ -28,5 +28,18 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
-    "x2_fields": "action_process_image_name"
+    "x2_fields": "action_process_image_name",
+    "Aliases": [
+        {
+          "cliName": "childprocess",
```
Comment (David-BMS) [inline Packs/CoreAlertFields/IncidentFields/incidentfield-Target_Process_Name.json:34]: Duplicate

File: Packs/AzureActiveDirectory/ReleaseNotes/1_2_2.md (hunk 16 above)
Comment (yaakovi) [reply Packs/AzureActiveDirectory/ReleaseNotes/1_2_2.md:3]: yes

**Review by David-BMS — APPROVED** [review]
